app/assets/javascripts/components/story/StoryDescription.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import Parser from 'html-react-parser'
+
+import StoryLink from 'components/story/StoryLink';
+
+const LOCAL_STORY_REGEXP = /(?!\s|\b)(#\d+)(?!\w)/g;
+
+class StoryDescription extends React.PureComponent {
+
+  constructor(props) {
+    super(props);
+
+    const description = Parser(props.description, { replace: domNode => {
+      if (!domNode.attribs || !domNode.attribs['data-story-id']) return;
+
+      const id = domNode.attribs['data-story-id'];
+      const story = props.linkedStories[id];
+      return ( <StoryLink story={story} key={id} /> );
+    }});
+
+    this.state = {
+      description: description,
+      isEmpty: (!props.description || props.description.length === 0)
```

Comment:
You can replace `props.description.length === 0` by `!props.description.length`.